Backend/app/dashboard/Lib/src/Domain/Action/Dashboard/Registration.hs

```diff
@@ -322,7 +351,8 @@ registerFleetOwner req mbPersonId = do
   merchantServerAccessCheck merchant
   let city' = fromMaybe merchant.defaultOperatingCity req.city
   merchantAccess <- DP.buildMerchantAccess fleetOwner.id merchant.id merchant.shortId city'
-  QP.create fleetOwner
+  let mbBoolVerified = not <$> merchant.requireAdminApprovalForFleetOnboarding
```

Comment:
lets use `Just` everywhere
`let mbBoolVerified = Just . not $ fromMaybe False merchant.requireAdminApprovalForFleetOnboarding`